src/pages/index.astro

```diff
@@ -1,10 +1,26 @@
 ---
-import { SearchLink } from "../components/react/SearchLink";
+import { getCollection } from "astro:content";
+import type { SearchableLink } from "../components/search/Search";
+import { Search } from "../components/search/Search";
 import Layout from "../layouts/Layout.astro";
+
+const episodes = await getCollection("episodes");
+
+const searchableLinks: SearchableLink[] = [];
+for (const episode of episodes) {
```

Comment:
cant this also be done in the loader_

Reply:
IMO it will be the wrong place to do that. This is not something related to the episode itself, it's a custom re-mapping of the data for the scope of the Search component

Reply:
also the types are not defined, you can reuse the types I defined in this file https://github.com/vikkio88/ppp-notes/pull/3/files#diff-7b4cc027ff9e5cee786c59e4c450a869335604267d062a22d5d27e9a387aac13L6
the Show/Podcast classes  will cover nicely those obj coming down  from the feed